aplib.net-demo/Assets/Scripts/Items/PickupableItem.cs

```diff
@@ -0,0 +1,36 @@
+using UnityEngine;
+
+namespace Assets.Scripts.Items
+{
+    /// <summary>
+    /// Attach this class to make object pickupable.
+    /// </summary>
+    public class PickupableItem : MonoBehaviour
+    {
+        private void OnCollisionEnter(Collision collision)
```

Comment:
Ik denk dat je in de toekomst misschien liever een trigger hebt dan een collision, dat maakt het makkelijker om een grotere pickup radius te hebben dan dat de prefab groot is (voelt beter gameplay-wise). Maar voor nu ziet het er prima uit!

Reply:
Goed punt! Ik zou het nu doen eigenlijk.

Reply:
Mm dat zou goed kunnen. Ik had nu voor collision gekozen omdat mijn prefab de hele tijd door de vloer viel als ik onTrigger aanzette op de prefab maar moet daar dan even naar kijken.

Reply:
Thanks! Ik ga dat even proberen.

Reply:
Het item hoeft geen collider te hebben. Als je de OnTrigger aan zet zou hij ook niet door de vloer moeten vallen (en doet hij denk ik ook niet als je geen Rigidbody hebt)